src/main.c

```diff
@@ -466,18 +469,22 @@ void LCUI_InitApp(LCUI_AppDriver app)
 	LCUICond_Init(&MainApp.loop_changed);
 	LCUIMutex_Init(&MainApp.loop_mutex);
 	LinkedList_Init(&MainApp.loops);
+	LCUIProfile_Init(&MainApp.profile);
+	LCUI_BindEvent(LCUI_SETTINGS_CHANGE, OnSettingsChangeEvent, NULL, NULL);
+	LCUI_ResetSettings();
```

Comment:
@jduo 

> I feel like event fired by using LCU_ResetSettings() while initializing LCUI is getting evaluated instead of the one fired during LCUI_ApplySettings(), but there should be no event handler bound at this time.

Use `Settings_Init()` instead of `LCUI_ResetSettings()`.